### DISCUSSION TITLE
Webring

### DISCUSSION THREAD

**Discussion by ghost** [sdy60-2022/.github#26] [General]
Τι σκέψεις υπάρχουν για τη κατασκευή του web ring;
Θα γίνει όπως στο παράδειγμα που δίνεται στην εκφώνηση [https://oompaloompas-webring.netlify.app/](https://oompaloompas-webring.netlify.app/) με χρήση του [webring](https://github.com/maxboeck/webring) και deployment στο netlify ή θα φτιάξουμε κάτι δικό μας;

**Comment by Katethoma** [comment]
Το webring που φαίνεται στο παράδειγμα φαίνεται καλό γιατί ο αριθμός των ατόμων που συμμετέχουν μοιάζει με τα μέλη του οργανισμού μας. Το θέμα είναι ο πίνακας μελών θα συνδεθεί όπως εμφανίζεται εκεί ;

**Comment by sotkot** [comment]
Έχω ενσωματώσει στο CV μου (που σερβίρεται μέσω GitHub) ό,τι χρειάζεται για να δοκιμάσουμε το netlify Webring. Οπότε προχωράμε κι αν κάτι δεν πάει καλά, το αλλάζουμε.
